.aspect/bazelrc/performance.bazelrc

```diff
@@ -18,3 +12,16 @@ build --experimental_reuse_sandbox_directories
 # author.
 # Docs: https://bazel.build/reference/command-line-reference#flag--legacy_external_runfiles
 build --nolegacy_external_runfiles
+
```

Comment:
how about just removing these presets and use bazelrc-preset.bzl ?

Reply:
I might end up opening another PR for the presets since this PR is failing way more then I was expecting :/